.github/workflows/main.yml

```diff
@@ -168,6 +168,19 @@ jobs:
       make-arguments: ${{ github.event.inputs.make-arguments }}
     if: needs.prepare.outputs.linux-x64 == 'true'
 
+  build-linux-x64-static-jdk:
```

Comment:
Move this closer to `build-linux-x64-static-libs`? These are somewhat related, at least in their need for `debug-levels: release`. Probably copy a comment about debug-level there.

Reply:
Also maybe just `build-linux-x64-static`? `-jdk` feels a bit redundant. I see you had it before, would you still prefer `...-static-jdk`?

Reply:
Thanks for the quick look, @shipilev! Also, thanks for separating the static-libs-bundle with https://github.com/openjdk/jdk/pull/23715. I haven't responded to your earlier comment as I wanted to wait for the GHA testing results.

I renamed to `...-static-jdk` to be more consistent with the new `...-static-libs` job. `build-linux-x64-static` still sounds fine to me, I'll rename it back.